docs/static_analysis.md

```diff
@@ -1,33 +1,296 @@
-Analysis
-========
+Solidity Static Analysis
+========================
 
-This section gives information about the last compilation. By default, a
-new analysis is run at each compilation.
+Static code analysis is a process to debug the code by examining it and without actually executing the code. 
 
-The analysis tab gives detailed information about the contract code. It
-can help you avoid code mistakes and to enforce best practices.
+`Solidity Static Analysis` plugin performs static analysis on Solidity smart contracts once they are compiled. It helps to check security vulnerabilities and bad development practices. This plugin comes with `Solidity` environment of Remix IDE. It can also be activated individually from `Plugin Manager`.
```

Comment:
edit: - the sentence that begins with "It helps to check..." I think could be: "It checks for check security vulnerabilities and bad development practices, among other issues.

Reply:
done